docs/py/Benchmarks.py

```diff
@@ -0,0 +1,64 @@
+import timeit
+import secrets
+import numpy
+import matplotlib.pyplot as plt
+from decimal import Decimal
+import DafnyVMC
+from IBM import sample_dgauss
+from datetime import datetime
+
+sigma_range = 100
+step = 1
+sigmas = numpy.arange(0.000001, sigma_range, step).tolist()
+
+vmc_mean = []
+vmc_std = []
+ibm_mean = []
+ibm_std = []
+
+fig,ax1 = plt.subplots()
+
+r = DafnyVMC.Random()
+
+for sigma in sigmas:
+    vmc = []
+    ibm = []
+    sigma_num, sigma_denom = Decimal(sigma).as_integer_ratio()
+    sigma_squared = sigma ** 2
+
+    for i in range(1100):
+        start_time = timeit.default_timer()
+        r.DiscreteGaussianSample(sigma_num, sigma_denom)
+        elapsed = timeit.default_timer() - start_time
+        vmc.append(elapsed)
+
+    for i in range(1100):
+        start_time = timeit.default_timer()
+        sample_dgauss(sigma_squared, rng=secrets.SystemRandom())
```

Comment:
You may want to extract the `rng` in its own variable before the loop, to avoid potential latency impact of `secrets.SystemRandom()` (although it may be negligible)